core/server/src/main/java/alluxio/worker/file/FileSystemWorker.java

```diff
@@ -60,20 +67,29 @@ public FileSystemWorker(BlockWorker blockWorker) throws IOException {
 
     mConf = WorkerContext.getConf();
     mFileDataManager = new FileDataManager(Preconditions.checkNotNull(blockWorker));
+    mUnderFileSystemManager = new UnderFileSystemManager();
 
     // Setup AbstractMasterClient
     mFileSystemMasterWorkerClient = new FileSystemMasterClient(
         NetworkAddressUtils.getConnectAddress(ServiceType.MASTER_RPC, mConf), mConf);
+
+    mServiceHandler = new FileSystemWorkerClientServiceHandler(this);
   }
 
-  /**
-   * {@inheritDoc}
-   * <p>
-   * {@link FileSystemWorker} exposes no RPC service.
-   */
   @Override
   public Map<String, TProcessor> getServices() {
-    return new HashMap<String, TProcessor>();
+    Map<String, TProcessor> services = new HashMap<>();
```

Comment:
I think now that we are on Java 7, we can start changing `Maps.newHashMap()` to `new HashMap<>()`, since it originally was a convenience method to avoid the verbosity of parameterizing the new hashmap? Are there other benefits to using `Maps.newHashMap`?